lld/ELF/ScriptParser.cpp

```diff
@@ -299,6 +304,15 @@ void ScriptParser::readDefsym(StringRef name) {
   script->sectionCommands.push_back(cmd);
 }
 
+void ScriptParser::readNoCrossRefs(bool to) {
+  expect("(");
+  NoCrossRefCommand cmd{{}, to};
+  while (!errorCount() && !consume(")"))
+    cmd.outputSections.push_back(unquote(next()));
+  if (cmd.outputSections.size() >= 2)
+    script->noCrossRefs.push_back(std::move(cmd));
+}
```

Comment:
Would it be worth a warning if the `outputSections.size() < 2` I don't think there could be a legal use case for `NOCROSSREFS()` or `NOCROSSREFS(<section>)`.

Reply:
Added a warning.